components/ai_chat/resources/untrusted_conversation_frame/components/assistant_response/tool_event.module.scss

```diff
@@ -17,11 +17,15 @@
   position: relative;
 
   .toolLabel {
+    all: unset;
     font: var(--leo-font-small-regular);
     color: var(--leo-color-text-secondary);
 
     &.isExpandable {
       cursor: pointer;
+      &:focus-visible {
+        outline: 1px solid var(--leo-color-schemes-outline);
```

Comment:
Is this right? Don't we normally do `box-shadow: var(--leo-effect-elevation-01)`

Reply:
hmm, that doesn't have enough contrast at all IMO, especially in dark mode